pulser-simulation/pulser_simulation/qutip_result.py

```diff
@@ -157,7 +157,7 @@ def get_state(
         is_density_matrix = state.isoper
         if ignore_global_phase and not is_density_matrix:
             full = state.full()
-            global_ph = float(np.angle(full[np.argmax(np.abs(full))]))
+            global_ph = float(np.angle(full[np.argmax(np.abs(full))])[0])
```

Comment:
Wow they made a major change

> Only ndim-0 arrays are treated as scalars. NumPy used to treat all arrays of size 1 (e.g., np.array([3.14])) as scalars. In the future, this will be limited to arrays of ndim 0 (e.g., np.array(3.14)). The following expressions will report a deprecation warning: